src/store/topic.ts

```diff
@@ -43,7 +43,7 @@ export class Topic implements TopicModel {
     @observable migratedFromJsonType: boolean;
     @observable schemaIdAwareSerializationEnabled: boolean;
     @observable contentType: ContentType;
-    @observable maxMessageSize = 10240;
+    @observable maxMessageSize = 102400;
```

Comment:
Can we use `@observable maxMessageSize = DEFAULT_TOPIC_VALUES.advancedValues.maxMessageSize` here?